taskcluster/ci/finetune-teacher/kind.yml

```diff
@@ -1,137 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I'm curious why `finetune-teacher` is going away. (Is there a doc or something else written about this that I can refer to for the high level bits?)

Reply:
It's going away together with `merge-augmented` because now we can specify what datasets to mix and how to split training stages in the OpusTrainer config: https://github.com/mozilla/firefox-translations-training/pull/219/files#diff-efcaa59d90628026042cd6413c703108f907081e7e617bd363d653c6e263e7a1. This simplifies the pipeline a bit. You can look at https://github.com/hplt-project/OpusTrainer readme.